web/views.py

```
@@ -6469,6 +6476,202 @@ def all_study_groups(request):
+            messages.success(request, "Your subscription has been reactivated.")
+        else:
+            messages.error(request, result["error"])
+
+    except Exception as e:
+        messages.error(request, str(e))
+
+    return redirect("membership_settings")
+
+
+@login_required
+def update_payment_method(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `update_payment_method`

(ANN201)